hack/chains/end-to-end-demo.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+#
+# A demo script that runs a build pipeline including HACBS specific tests and a sample release pipeline with HACBS Enterprise Contract test
+
+APP_GIT_REPOSITORY=https://github.com/jduimovich/single-nodejs-app
+APP_BUILDER=nodejs-builder
+APP_IMAGE_NAME="${APP_GIT_REPOSITORY/*\/}"
+APP_IMAGE_REF="quay.io/${MY_QUAY_USER}/${APP_IMAGE_NAME}"
+
+# Check for $MY_QUAY_USER
+if [[ -z ${MY_QUAY_USER+x} ]]; then
```

Comment:
I don't understand the +x here.

Reply:
I didn't either, see https://stackoverflow.com/a/13864829/225448